.pre-commit-config.yaml

```
@@ -1000,7 +1000,7 @@ repos:
         name: Run mypy for providers
         language: python
         entry: ./scripts/ci/pre_commit/pre_commit_mypy.py --namespace-packages
-        files: ^airflow/providers/.*\.py$|^tests/providers/\*\.py$|^tests/system/providers/\*\.py$
+        files: ^airflow/providers/.*\.py$|^tests/providers/.*\.py$|^tests/system/providers/.*\.py$
```

```suggestion
        files: ^airflow/providers/.*\.py$|^tests/providers/.+\.py$|^tests/system/providers/.+\.py$
```

Doesn’t really matter in practice but I don’t think we want to allow a file named exactly `.py`

---

Well.. As we see here - seems we already had trouble with getting the regexp right :) .  I do not want to complicate it even more.... You know the saying "Have a problem? introduce regexp, now you have two problems..."